### PR TITLE
Competitive Programming : : Sliding Window Max

### LINKED ISSUE #1217: Competitive Programming : : Sliding window maximum (documentation)
<hr>

## Description 📜

I would like to provide documentation about the Sliding window maximum problem.

<hr>

## Domain of Contribution 📊

- [x] Competitive Programming

<hr>

### PR DESCRIPTION
**Pull request by areenoverclouds**
<hr>

## Description 📜

I have added the documentation of the Sliding Window Maximum Problem.
Topic: Maximum of all subarrays of given size K.
Language: C++
Format: Documentation 
Fixes #1217
<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Competitive Programming

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

File: SlidingWindow_cpp.ipynb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZsZ5UUMT56to"
+      },
+      "source": [
+        "**Sliding Window Maximum (Maximum of all subarrays of size k)**\n",
+        "\n",
+        "Language : C++\n",
```
Comment (ravireddy07) [inline SlidingWindow_cpp.ipynb:40]: No need to specify language

File: SlidingWindow_cpp.ipynb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        "\n",
+        "Time Complexity : O(n*k)\n",
+        "\n",
+        "Space Complexity : O(1)\n",
+        "\n",
```
Comment (ravireddy07) [inline SlidingWindow_cpp.ipynb:45]: Specify Complexity with a Title at the bottom

File: SlidingWindow_cpp.ipynb [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        "    return 0;\n",
+        "}\n",
+        "```\n",
+        "\n",
+        "\n",
+        " \n"
+      ]
+    }
+  ]
+}
```
Comment (ravireddy07) [inline SlidingWindow_cpp.ipynb:93]: leave an empty line at the end of the file

**Review by ravireddy07 — CHANGES_REQUESTED** [review]
Please Explain you Problem and approaches.
And explain you solution in words.

File: SlidingWindow_cpp.ipynb (hunk 2 above)
Comment (areenoverclouds) [reply SlidingWindow_cpp.ipynb:45]: Sir I have resolved the issues and opened a new pull request #3614
Sorry for the inconvenience.

Comment (ravireddy07) [reply SlidingWindow_cpp.ipynb:45]: you can resolve and update this PR also. no need of creating a new PR.

**Pull request closed without merging**